chaco/tests/test_scatter_1d.py

```diff
@@ -0,0 +1,139 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+"""
+Unit tests for scatter_1d function
+"""
+
+from numpy import sort
+from numpy.random import random, randint, uniform
+import unittest
+from chaco.api import ArrayPlotData, Plot
+
+
+class Scatter1DTestCase(unittest.TestCase):
+    # test the default value 4.0
```

Comment:
that comment line can be removed in favour of a clear method name or moved as a docstring to the test method.